synapseclient/core/download/download_functions.py

```diff
@@ -433,7 +489,15 @@ async def download_by_file_handle(
             concrete_type = file_handle["concreteType"]
             storage_location_id = file_handle.get("storageLocationId")
 
+            actual_file_size = file_handle.get("contentSize", 0)
+            actual_md5 = file_handle.get("contentMd5")
+
+            if actual_file_size:
+                span.set_attribute("synapse.file.size_bytes", actual_file_size)
+
             if concrete_type == concrete_types.EXTERNAL_OBJECT_STORE_FILE_HANDLE:
+                span.set_attribute("synapse.storage.provider", "s3")
```

Comment:
Just wanted to check where we would be tracking the "google" buckets

Reply:
Good question, I'll need to revist what code path Google buckets follow. This path is specifically for user owned s3 bucket downloads.